x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/action/role/PutRoleRequestTests.java

```diff
@@ -41,7 +41,7 @@
     public void testValidationErrorWithUnknownClusterPrivilegeName() {
         final PutRoleRequest request = new PutRoleRequest();
         request.name(randomAlphaOfLengthBetween(4, 9));
-        String unknownClusterPrivilegeName = randomAlphaOfLengthBetween(7, 9);
+        String unknownClusterPrivilegeName = "-unknown-";
```

Comment:
```suggestion
        String unknownClusterPrivilegeName = "unknown_" + randomAlphaOfLengthBetween(3, 9);
```
Random was good, plain alpha was good, it just had a chance of doing the wrong thing.
I don't think a leading `-` is good because that's not representative of a real typo.